library/books-api-v2/v2/Author.ts

```diff
@@ -0,0 +1,4 @@
+export interface Author {
+    name: string;
+    birthDate: Date,
```

Comment:
Honestly, I would strongly consider just doing `birthYear: number`. JS `Date`s as a whole are tricky, especially when converting _from_ JSON. To the point that I actually hesitate giving students an example of just using them flatly even in a read-only API.